sktime/datatypes/_adapter/polars.py

```diff
@@ -252,14 +252,6 @@ def check_polars_frame(
             else:
                 metadata["n_instances"] = "NA"
 
-    if scitype in ["Panel", "Hierarchical"]:
```

Comment:
why are we removing this?

Reply:
is `check_polars_frame` called from anywhere else? Just checking. If not, should be an ok refactor move.

Reply:
Yes its not called anywhere else